srcStatic/GameModules/IniModule.cpp

```diff
@@ -9,7 +10,10 @@ IniModule::IniModule(IniSection iniSection)
 {
 	try {
 		// Get the DLL name from the corresponding section
-		LoadModuleDll(DllName());
+		const fs::path path = DllName();
+		if (path.empty() == false) {
+			LoadModuleDll((path.is_absolute() ? path : GetOpuDirectory() / path).string());
```

Comment:
@DanRStevens, I'm not sure. There is a `AppendSubDirectory` in OP2Utility, although I'm not sure it is a well thought out method that I wrote. I think this line of code is fairly clear in intent without modification. 

I think the important thing here is to make sure the current environment directory isn't used, especially if using a debugger because it may not be coincident with the actual Outpost 2 directory. And I think this has been taken into account.